### ISSUE TITLE
Reformat output.

### ISSUE THREAD

**Issue by cowboy** [gruntjs/grunt-contrib-jshint#162]
This is what I see now:

![](http://benalman.com/grab/0c3c16.png)

Compare that to >=0.8.0:

![](http://benalman.com/grab/df41d6.png)

While I like the line number on the left and the reduction of colors in general, I don't really like the indentation of the `^ message` because it's going to wrap oddly in many circumstances.

First, I'd remove the per-file indentation, and would intelligently compute the width of the "line number" column per-file using the [grunt.log.table](http://gruntjs.com/api/grunt.log#grunt.log.table) method, like so:

```
$ grunt
Running "jshint:all" (jshint) task

Gruntfile.js
20 |    (function() { /* need to test indentation and wrapping */ }(x));
                                                                    ^ 'x' is not defined.
index.js
211 |log.verbose = {};
     ^ 'log' is not defined.
212 |log.notverbose = {};
     ^ 'log' is not defined.
215 |Object.keys(log).filter(function(key) {
                 ^ 'log' is not defined.
```

Maybe after that, I'd go back to visually highlighting the invalid character and not indenting the warning message, simulated here with `[]`.

```
$ grunt
Running "jshint:all" (jshint) task

Gruntfile.js
20 |    (function() { /* need to test indentation and wrapping */ }([x]));
    'x' is not defined.
index.js
211 |[l]og.verbose = {};
     'log' is not defined.
212 |[l]og.notverbose = {};
     'log' is not defined.
215 |Object.keys([l]og).filter(function(key) {
     'log' is not defined.
```

Basically, I'd like to see a hybrid between the old and the new way, because I don't think the new way is a clear-cut improvement.

**Comment by acostaf** [comment]
+1
